public/static/docs/user-guide/external-dependencies.md

```diff
@@ -185,6 +185,6 @@ outs:
 ```
 
 The `url` and `rev_lock` subfields under `repo` are used to save the origin and
-version of the dependency.
+[revision](https://git-scm.com/docs/revisions) of the dependency, respectively.
```

Comment:
same here, version is probably fine and more appropriate unless we change the whole sentence even futhether

Reply:
Agree. Changed here and in `public/static/docs/get-started/import-data.md`.